tests/main/menu-spec.ts

```diff
@@ -127,14 +127,47 @@ describe('menu', () => {
         IpcEvents.SELECT_ALL_IN_EDITOR,
       );
     });
+
+    describe('setup Show Me menu', () => {
+      it('show me menu correctly contains no check when no template is selected', () => {
+        overridePlatform('darwin');
+        setupMenu({});
+
+        const result = (electron.Menu.buildFromTemplate as any).mock
+          .calls[0][0];
+        const showMeMenu = result[result.length - 2];
+
+        let showMeItemChecked = true;
+        if (showMeMenu.submenu[0].submenu[0]) {
+          showMeItemChecked = showMeMenu.submenu[0].submenu[0].checked;
+        }
+        expect(showMeItemChecked).toEqual(false);
```

Comment:
This can be simplified to be less verbose:

```suggestion
        const showMeItemChecked = showMeMenu?.submenu[0]?.submenu[0]?.checked;
        expect(showMeItemChecked).toBe(false);
```

The biggest change here is the use of the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (`?.`) to drill into possibly undefined object properties.

It's also unnecessary to set the variable to `true` initially because `showMeItemChecked` will be `undefined` if the optional chaining fails, and `expect(undefined).toBe(false)` will throw.